global/meta/models.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # Types:
-#     - Nativ datatypes: string, number, boolean, JSON
+#     - Nativ datatypes: string, text, number, boolean, JSON
```

Comment:
Could you add a short explanation what the different between string and text is?

Reply:
Only in backend: The string is limited to 256 char, text is unlimited